bundle/filefs.go

```diff
@@ -0,0 +1,73 @@
+// +build go1.16
+
+package bundle
+
+import (
+	"fmt"
+	"io"
+	"io/fs"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+type dirLoaderFS struct {
+	filesystem fs.FS
+	files      []string
+	idx        int
+}
+
+// Interface guard for dirLoaderFS to validate that it implements the DirectoryLoader interface
+var _ DirectoryLoader = (*dirLoader)(nil)
```

Comment:
This is fine, but is it necessary? The function `NewFSLoader`, returning a `*dirLoader` for the DirectoryLoader interface gives us the same assurance, I think.

Reply:
No, you are 100% correct and this isn't neccessary :-)